pkg/model/application.go

```diff
@@ -81,3 +81,11 @@ func (a *Application) IsOutOfSync() bool {
 func IsApplicationConfigFile(filename string) bool {
 	return filename == DefaultApplicationConfigFilename || strings.HasSuffix(filename, applicationConfigFileExtention)
 }
+
+func ToApplicationKind(k string) (ApplicationKind, bool) {
+	appKind, ok := ApplicationKind_value[strings.ToUpper(string(k))]
```

Comment:
I don't think we should allow both upper case and lower case.
Exactly matching is fine to me.